bl31/aarch64/bl31_entrypoint.S

```diff
@@ -42,6 +43,39 @@
 	 */
 
 func bl31_entrypoint
+#ifdef BL31_ENABLE_RELOCATION
```

Comment:
As discussed in https://github.com/ARM-software/tf-issues/issues/382, BL31_DIFFERENT_LOAD_RUN_ADDRESS is a less confusing name